ExtraDry/ExtraDry.Core/Core/PropertySize.cs

```diff
@@ -1,11 +1,16 @@
-﻿namespace ExtraDry.Blazor;
+﻿namespace ExtraDry.Core;
 
 /// <summary>
 /// A semantic description of the size of a property, used to determine how to layout forms.
 /// </summary>
 [JsonConverter(typeof(JsonStringEnumConverter))]
 public enum PropertySize {
 
+    /// <summary>
+    /// Size is calculated based on the property details.
+    /// </summary>
+    Calculated = 0,
```

Comment:
per above comment, this is then obsolete.  In particular, I don't want users to be able to manually set the property size to 'calculated'.

Reply:
This has been removed following the changes to the `Size` property.